src/confab/agent/runner_test.go

```diff
@@ -99,7 +98,7 @@ var _ = Describe("Runner", func() {
 					},
 					{
 						Action: "agent-runner.cleanup.remove.failed",
-						Error:  expectedError,
+						Error:  errors.New(err.Error()),
```

Comment:
`Error: errors.New(err.Error())`: Is there a reason this can't be just `Error: err`